pkg/kopialib/storage/errors.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+var (
+	ErrMissingRequiredFieldMsg      = "missing required field: %s for storage provider %s"
+	ErrStorageOptionsCannotBeNilMsg = "storage options cannot be nil for the provider %s"
```

Comment:
```suggestion
	MissingRequiredFieldMsg      = "missing required field: %s for storage provider %s"
	StorageOptionsCannotBeNilMsg = "storage options cannot be nil for the provider %s"
```

Should we do this?
Because these are not actually errors but just messages?

Reply:
Done !